### PR TITLE
Implment GetPageSource in wxWebViewChromium3

### PR DESCRIPTION
**Pull request by hokein**
Please review it, Thanks.

### CODE REVIEW

File: webview_chromium3.h [hunk 1]
```diff
@@ -232,6 +234,9 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewChromium : public wxWebView
     //The timer calls the CEF event loop
     wxTimer *m_timer;
 
+    // Current main frame page source
+    wxString m_page_source;
```
Comment (sjlamerton) [inline webview_chromium3.h]: For consistency with other wxWidgets code this should probably be m_pageSource

File: webview_chromium3.cpp [hunk 2]
```diff
@@ -102,6 +116,11 @@ void wxWebViewChromium::OnSize(wxSizeEvent& event)
     event.Skip();
 }
 
+void wxWebViewChromium::setPageSource(const wxString& page_source)
```
Comment (sjlamerton) [inline webview_chromium3.cpp]: To keep it consistent with the other code could you rename this to SetPageSource?

**Comment by sjlamerton** [comment]
Looks good, will test once the name changes are made and then commit

File: webview_chromium3.h (hunk 1 above)
Comment (hokein) [inline webview_chromium3.h]: Done.

File: webview_chromium3.cpp (hunk 2 above)
Comment (hokein) [inline webview_chromium3.cpp]: Done.

**Comment by sjlamerton** [comment]
Thanks for the changes, worked fine when I was testing.

**Pull request merged**